src/metabase/api/collection.clj

```diff
@@ -876,38 +880,50 @@
 (api/defendpoint GET "/graph"
   "Fetch a graph of all Collection Permissions."
   [namespace]
-  {namespace (s/maybe su/NonBlankString)}
+  ^:malli {namespace [:maybe [:string {:min 1}]]}
   (api/check-superuser)
```

Comment:
The way you signal that the type checking is done with malli by giving this map this metadata.